blog/_posts/2023-07-21-2023-nextstrain-hiring.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Openings for bioinformatician and software engineer to contribute to Nextstrain platform
+author: Trevor Bedford
+---
+
+Positions for a __bioinformatician__ and a __software engineer__ are available immediately in the Bedford lab at the Fred Hutch. Details for both positions follow:
```

Comment:
Could we use the full name of the bioinformatics position here and also link each title to the corresponding job descriptions?

```markdown
Positions for [a bioinformatics analyst](https://careers-fhcrc.icims.com/jobs/25697/job) and [a software engineer](https://hhmi.wd1.myworkdayjobs.com/en-US/External/job/Software-Engineer-II--Bedford-Lab_R-2176) are available immediately in the Bedford lab at the Fred Hutch. Details for both positions follow:
```

The links also serve to emphasize the titles without additional bolding, but we could keep the bold if you like.

Reply:
Good idea! I [updated](https://github.com/blab/blotter/compare/b783d06843306c37ec7e376f70adb50873ee262a..b75e205e7dfe295129d9584f160ed003d5292358) with links to the headings rather than external links, since those are provided at the bottom of each already. This should encourage jumping to the sections individually, which helps your point in https://github.com/blab/blotter/pull/34#discussion_r1271149591.